src/main/java/io/supertokens/webserver/api/session/RefreshSessionAPI.java

```diff
@@ -62,6 +62,11 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             SessionInformationHolder sessionInfo = Session.refreshSession(main, refreshToken, antiCsrfToken,
                     enableAntiCsrf);
             JsonObject result = new JsonParser().parse(new Gson().toJson(sessionInfo)).getAsJsonObject();
+
+            if (!super.getVersionFromRequest(req).equals("2.13")) {
```

Comment:
You don't want to do `.equals`. You want to check if it's >= 2.13. Cause if we add 2.14 later, this will be false and then session grant will be removed breaking everything.